src/cruds/device.py

```diff
@@ -246,6 +246,48 @@ async def get_historical_device_data_alarm(db: AsyncSession, device_id: str) ->
     return result
 
 
+async def get_device_data_notifications(db: AsyncSession, device_id: str) -> List[device_schema.DeviceNotificationData]:
+    """
+    Get notifications data from db
+
+    Args:
+        db (AsyncSession): AsyncSession
+        device_id (str): Device id
+
+    Returns:
+        [device.schema.DeviceNotificationData]: List of device historical notification data
+    """
+    stmt = """
+        SELECT
+            CONTENT_TYPE,
```

Comment:
Can you please add `ID`? We need to update the `IS_READ` when user click the message.